src/bank02_player_menus.asm

```diff
@@ -2488,9 +2490,21 @@ jp_02_51fb:
     ld   A, [wMoneyLow]                                ;; 02:5218 $fa $be $d7
     ld   E, A                                          ;; 02:521b $5f
     add  HL, DE                                        ;; 02:521c $19
-    jr   NC, .jr_02_5222                               ;; 02:521d $30 $03
-    ld   HL, rIE                                       ;; 02:521f $21 $ff $ff
-.jr_02_5222:
+    push HL
+    ld   HL, wStatusEffect
+    jr   NC, .set_1
+    set  7, [HL]
+.set_1:
+    bit  7, [HL]
+    pop  HL
+    jr   Z, .set_2
+    ld   A, L
+    sub  A, $9f
+    ld   A, H
+    sbc  A, $86
+    jr   C, .set_2
+    ld   HL, $869f
+.set_2:
```

Comment:
Any reasons for or against making this call `addMoney`?

Reply:
Looks like `addMoney` does a call to `drawMoneyOnStatusBarTrampoline` that *might* draw over the active windows. If so, an easy solution would be to change `addMoney` to have two variations.

`addMoney:
call _addMoney
jp drawMoneyOnStatusBarTrampoline

_addMoney:
...
; call drawMoneyOnStatusBarTrampoline
ret`

Reply:
It's using addMoneyAdjustValues now :)